scripts/check_go_integration.sh

```diff
@@ -9,7 +9,8 @@ set -e
 echo "Running aries-framework-go integration tests..."
 PWD=`pwd`
 cd test/bdd
-go test -count=1 -v -cover . -p 1 -timeout=20m -race
+#go test -count=1 -v -cover . -p 1 -timeout=20m -race
```

Comment:
leftover.

Reply:
it was temporary - see https://github.com/hyperledger/aries-framework-go/pull/1553#issuecomment-607845970